apps/meteor/server/models/raw/LivechatInquiry.ts

```diff
@@ -101,6 +101,12 @@ export class LivechatInquiryRaw extends BaseRaw<ILivechatInquiryRecord> implemen
 				},
 				sparse: true,
 			},
+			{
+				key: {
+					lockedAt: 1,
+				},
+				sparse: true,
+			},
```

Comment:
We can presume that, but it should involve more testing since this code has quite some time around. `locked` and `lockedAt` are set at the exact same time and there's just one place that does the call.

However i'd say removing that should have more testing involved and even since it's not a breaking change i'd prefer to have that change (if done) on v7 instead than on a patch.

(should be pretty much a safe change but just in case)

Reply:
Updated PR to remove the index and the need for the $or. I expect almost same perf as adding the index.

Great suggestion! Thanks man